deltachat-ffi/deltachat.h

```diff
@@ -4882,7 +4910,44 @@ uint32_t        dc_lot_get_id            (const dc_lot_t* lot);
  * @param lot The lot object.
  * @return The timestamp as defined by the creator of the object. 0 if there is not timestamp or on errors.
  */
-int64_t          dc_lot_get_timestamp     (const dc_lot_t* lot);
+int64_t         dc_lot_get_timestamp     (const dc_lot_t* lot);
+
+
+/**
```

Comment:
```suggestion
/**
 * @class dc_reactions_t
 *
 * An object representing all reactions for a single message.
 */

/**
```

needed to let Doxygen create an overview page.